lgc/test/CsComputeLibraryPayload.lgc

```diff
@@ -12,16 +12,14 @@ target triple = "amdgcn--amdpal"
 define spir_func <10 x i32> @func(<10 x i32> %arg) local_unnamed_addr #0 !lgc.shaderstage !7 {
 ; CHECK-LABEL: func:
 ; CHECK: s_waitcnt vmcnt(0) expcnt(0) lgkmcnt(0)
-; CHECK-NEXT: s_waitcnt_vscnt null, 0x0
```

Comment:
The changes in this file are designed to work either with or without the LLVM patch.